src/discord/handlers/MessageHandler.js

```diff
@@ -5,19 +5,23 @@ class MessageHandler {
   }
 
   async onMessage(message) {
-    if (!this.shouldBroadcastMessage(message)) {
-      return
-    }
 
     if (this.command.handle(message)) {
       return
     }
 
+    if (!this.shouldBroadcastMessage(message)) {
+      return
+    }
```

Comment:
This is checking if:
- The message author is not a bot
- The message's channel is the designated guild chat channel
- The message has content and the content has length

Moving it down below handling the message as a command only changes it so that commands can be run anywhere, which has already been rejected in #16:
> On the note of using commands globally everywhere, it was a conscious discussion to not allow the bot commands to be used outside of the channel the bot is linked with since most servers using the bridge bot will also have other bots in their server, and the command prefix might conflict with the other bots, if the system is changed so commands can be used throughout the entire Discord server the system would have to be configurable, so people self-hosting the bot can choose to turn it on or off.

Currently this implementation means other bots could run commands and there could be crashes as messages without content could be handled as if there was content there.